client/extensions/woocommerce/app/order/order-details/row-shipping.js

```diff
@@ -26,16 +27,16 @@ class OrderShippingRow extends Component {
 		}
 
 		const tax = (
-			<div className="order__details-totals-tax">
+			<div className="order-details__totals-tax">
 				{ formatCurrency( order.shipping_tax, order.currency ) }
 			</div>
 		);
 
 		return (
-			<div className="order__details-total-shipping">
-				<div className="order__details-totals-label">{ translate( 'Shipping' ) }</div>
+			<div className="order-details__total-shipping">
+				<div className="order-details__totals-label">{ translate( 'Shipping' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Shipping address' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).